tests/PHPUnit/WURFL/Request/UserAgentNormalizer/AndroidTest.php

```diff
@@ -30,11 +30,11 @@ function normalizerDataProvider() {
 		return array(
 			array("FOO", "FOO"),
 			array("Mozilla/5.0 (Linux; U; Android 1.0.15; fr-fr; A70HB Build/CUPCAKE) AppleWebKit/525.10+ (KHTML, like Gecko) Version/3.0.4 Mobile Safari/523.12.2",
-				"1.0 A70HB".WURFL_Constants::RIS_DELIMITER."Mozilla/5.0 (Linux; U; Android 1.0.15; xx-xx; A70HB Build/CUPCAKE) AppleWebKit/525.10+ (KHTML, like Gecko) Version/3.0.4 Mobile Safari/523.12.2"),
```

Comment:
Please check if reverting the changes in this file will make the unittests pass.